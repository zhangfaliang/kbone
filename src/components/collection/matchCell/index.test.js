import toJson from 'enzyme-to-json';
import { render, mount } from 'enzyme';
import MatchCell from './index';
import moment from 'moment';

jest.useFakeTimers();
describe('CountDown', async () => {
  it('Button ', () => {
    expect.assertions(1);
    const wrapper = render(<MatchCell />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('leftcls  ture', () => {
    const wrapper = mount(<MatchCell leftcls={true} live={true} />);
    wrapper.find('div.left-name');
  });
});
