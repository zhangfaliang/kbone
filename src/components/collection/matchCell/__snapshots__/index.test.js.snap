// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountDown Button  1`] = `
<div
  class="default-cell"
>
  <div
    class="false"
  />
  <div />
  <div>
    <div
      class="leftText"
    >
      <div />
    </div>
  </div>
</div>
`;
